JsonConfig/Config.cs

```diff
@@ -119,18 +119,42 @@ static Config ()
 		private static FileSystemWatcher userConfigWatcher;
 		public static void WatchUserConfig (FileInfo info)
 		{
+            var lastRead = File.GetLastWriteTime(info.FullName);
```

Comment:
indentation is broken with your changes, could you re-indent and resubmit the PR again?